05/DB.test.js

```diff
@@ -1 +1,21 @@
-import DB from './DB';
\ No newline at end of file
+import DB from "./DB";
+describe("DB", () => {
+	describe("Method insert()", () => {
+		it("return error when is not a number", async () => {
+			const db = new DB();
```

Comment:
Musisz pamiętać o `expect.assertions(1);` bo kod, który piszesz jest warunkowy - nie zawsze jest błąd.
My musimy wskazać, że spodziewamy się jednego porównania (1) jeśli nie wystąpi to ma być błąd.